README.md

```diff
@@ -109,7 +109,7 @@ Only when you clear the data directory in the CKB node, or switch to a different
 To play with wallets, first we need to add some capacities to a wallet:
 
 ```bash
-[1] pry(main)> bob = Ckb::Wallet.from_hex(api, "e79f3207ea4980b7fed79956d5934249ceac4751a4fae01a0f7c4a96884bc4e3")
+[1] pry(main)> bob = Ckb::Wallet.from_hex(api, "0xe79f3207ea4980b7fed79956d5934249ceac4751a4fae01a0f7c4a96884bc4e3")
```

Comment:
Is the hex string parameter actually a private key? If so, perhaps this is good chance to rename it to `Wallet.from_private_key`?

Reply:
Yes, it a private key hex string, `new` requires a `Ckb::Key` object and `from_hex` requires a hex format string.